python/mleap/sklearn/preprocessing/data.py

```diff
@@ -532,11 +533,19 @@ def __init__(self):
 
     def serialize_to_bundle(self, transformer, path, model_name):
 
-        # compile tuples of model attributes to serialize
+        if transformer.strategy == 'most_frequent' or transformer.strategy == 'constant':
```

Comment:
I think we could just set `surrogate_value` if we wanted to implement the `constant_strategy`, because it looks like strategy is ignored during imputation and it just writes whatever constant surrogate value was in the mleap bundle:

https://github.com/combust/mleap/blob/d276c1c800277a1edcf30dda6e8559c7413b21f8/mleap-core/src/main/scala/ml/combust/mleap/core/feature/ImputerModel.scala#L18

But I'm not sure how this would interact with the spark extensions because `constant` is not a valid strategy there:

https://github.com/combust/mleap/blob/fb7493cc662a39eaac5e9bb473f7ea7df71d016a/mleap-spark-extension/src/main/scala/org/apache/spark/ml/mleap/feature/Imputer.scala#L121